src/payment/strategies/googlepay/googlepay-braintree-initializer.ts

```diff
@@ -79,7 +81,7 @@ export default class GooglePayBraintreeInitializer implements GooglePayInitializ
             transactionInfo: {
                 currencyCode: checkout.cart.currency.code,
                 totalPriceStatus: 'FINAL',
-                totalPrice: checkout.grandTotal.toString(),
+                totalPrice: round(checkout.grandTotal, 2).toString(),
```

Comment:
Can I ask, why do you need to manually do the rounding? The grand total amount should already be rounded according to the current settings configured by the merchant.